src/main/kotlin/org/opensearch/replication/ReplicationPlugin.kt

```diff
@@ -170,6 +170,8 @@ internal class ReplicationPlugin : Plugin(), ActionPlugin, PersistentTaskPlugin,
         val REPLICATION_METADATA_SYNC_INTERVAL = Setting.timeSetting("plugins.replication.follower.metadata_sync_interval",
                 TimeValue.timeValueSeconds(60), TimeValue.timeValueSeconds(5),
                 Setting.Property.Dynamic, Setting.Property.NodeScope)
+        val REPLICATION_RETENTION_LEASE_MAX_DURATION = Setting.timeSetting ("plugins.replication.follower.retention_lease_max_duration", TimeValue.timeValueHours(1), TimeValue.timeValueSeconds(1),
```

Comment:
can we make  max duration  as 12 hours ? 

Also rename to `retention_lease_failure_duration` to avoid confusion?

Reply:
you mean 12 hours as default? setting max allowed value is 12 hours right now.

I'll rename the setting name